app/client/src/components/ResultsTable.vue

```diff
@@ -0,0 +1,134 @@
+            {{sample.Cluster}}
+          </td>
+          <td v-if="sample.Rowspan !== 0"
+          style="vertical-align : middle;"
+          :rowspan="sample.Rowspan"
+          :class="(sample.Microreact === '\u2714')? 'checkmark' : 'processing'">
+            {{sample.Microreact}}
+          </td>
+          <td v-if="sample.Rowspan !== 0"
+          style="vertical-align : middle;"
+          :rowspan="sample.Rowspan"
+          :class="(sample.Network === '\u2714') ? 'checkmark' : 'processing'">
+            {{sample.Network}}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import { mapState } from 'vuex';
+import { VBTooltip } from 'bootstrap-vue-3';
+import { addRowspan, getRGB, verbalProb } from '../utils';
+
+export default defineComponent({
+  name: 'ResultsTable',
+  directives: {
+    'b-tooltip': VBTooltip,
+  },
+  data() {
+    return {
+      tableData: this.generateTableData(),
+    };
+  },
+  methods: {
+    getRGB,
+    verbalProb,
+    getTooltipText(data: Record<string, number>) {
+      const firstLine = 'Probability of resistance to:';
+      const pen = ['Penicillin: ', this.verbalProb(data.Penicillin, 'Penicillin'), ' <small>(', data.Penicillin, ')</small>'].join('');
+      const chlor = ['Chloramphenicol: ', this.verbalProb(data.Chloramphenicol, 'Chloramphenicol'), ' <small>(', data.Chloramphenicol, ')</small>'].join('');
+      const ery = ['Erythromycin: ', this.verbalProb(data.Erythromycin, 'Erythromycin'), ' <small>(', data.Erythromycin, ')</small>'].join('');
+      const tetra = ['Tetracycline: ', this.verbalProb(data.Tetracycline, 'Tetracycline'), ' <small>(', data.Tetracycline, ')</small>'].join('');
+      const cotrim = ['Cotrim: ', this.verbalProb(data.Trim_sulfa, 'Cotrim'), ' <small>(', data.Trim_sulfa, ')</small>'].join('');
+      return [firstLine, pen, chlor, ery, tetra, cotrim].join('<br/>');
+    },
+    getCluster(sample: string) {
+      return (this.results.perIsolate[sample].cluster
+        ? this.results.perIsolate[sample].cluster : this.analysisStatus.assign);
+    },
+    getMicroreact() {
+      return (this.analysisStatus.microreact === 'finished') ? '\u2714' : this.analysisStatus.microreact;
+    },
+    getNetwork() {
+      return (this.analysisStatus.network === 'finished') ? '\u2714' : this.analysisStatus.network;
+    },
+    generateTableData() {
+      if (this.results) {
+        const samples = this.results.perIsolate;
+        const items: Array<Record<string, string>> = [];
+        Object.keys(samples).forEach((sample) => {
+          items.push({
+            Hash: samples[sample].hash,
+            Filename: samples[sample].filename,
+            Sketch: (samples[sample].sketch) ? '\u2714' : 'processing',
+            AMR: samples[sample].amr ? samples[sample].amr : 'processing',
+            Cluster: this.submitStatus === 'submitted' ? this.getCluster(sample) : '',
```

Comment:
One issue here is that if the user submits an analysis but then adds more fasta files, the new files show up with ticks even though they haven't been analysed yet. I don't know what the desired UI is here: I guess the easiest thing would be to reset the analysis when new files are added, but probably the better thing would be to have file level statuses that store whether a particular file has been analysed, so that new files can be added and then submitted for analysis.

To replicate: 1. add fasta files 2. press "start analysis" 3. add more fasta files

Having individual file statuses would also allow the progress bar to be more fine-grained, could increment it every time a single file's status was updated rather than it having to wait for all files to be processed to progress.

Reply:
Since the analyses in the backend (cluster, microreact & network) do run with all files at once in a batch, adding new files later would mean we would have to rerun everything with all files (can't just combine old result with new one). So it is not intended to add more files after clicking the submit button. In the mockups the view was shifted after hitting submit, so that the dropzone would disappear and no further file submission would be possible. I have added that to the conditional rendering of the dropzone component that it is not available after submission.